js/script_pricelist.js

```diff
@@ -0,0 +1,530 @@
+
+    const makeup = {
```

Comment:
indentation is off. if you are using vs code try running `format document` command. On mac it <Command>+<Shift>+<p> will open the command input field and then you can type `format document` it might prompt you to install an extension to do so. Just pick the most popular one.